doc/spark_on_ray.md

```diff
@@ -60,3 +60,6 @@ spark = raydp.init_spark(...,enable_hive=True)
 spark.sql("select * from db.xxx").show()
 ```
 
+### Logging
++ Driver Log: By default, the spark driver log level is WARN. After getting a Spark session by running `spark = raydp.init_spark`, you can change the log level for example `spark.sparkContext.setLogLevel("INFO")`. You will also see some AppMaster INFO logs on the driver. This is because Ray redirects the actor logs to driver by default. To disable logging to driver, you can set it in Ray init `ray.init(log_to_driver=False)`
```

Comment:
I have some second thoughts on suggesting user to `log_to_driver` though, one of the biggest benefit on using raydp is I can print in spark functions and see the logs in driver, but setting `log_to_driver` will disable that all together. Is there a way to just disable to app master info log so we get a better notebook experience? @carsonwang

Reply:
Thanks for the suggestion, @pang-wu . We will take a look how to support it.